src/content-helper/editor-sidebar/title-suggestions/component.tsx

```diff
@@ -153,11 +170,19 @@ export const TitleSuggestionsPanel = (): JSX.Element => {
 	};
 
 	const parselyAISettings = <TitleSuggestionsSettings
-		tone={ tone }
-		persona={ persona }
-		onToneChange={ ( selectedTone ) => setTone( selectedTone ) }
-		onPersonaChange={ ( selectedPersona ) => setPersona( selectedPersona ) }
 		isLoading={ loading }
+		isOpen={ initialSettingsOpen }
+		onPersonaChange={ ( selectedPersona ) => {
+			onSettingChange( 'TitleSuggestionsPersona', selectedPersona );
+			setPersona( selectedPersona );
+		} }
+		onSettingChange={ onSettingChange }
+		onToneChange={ ( selectedTone ) => {
+			onSettingChange( 'TitleSuggestionsTone', selectedTone );
+			setTone( selectedTone );
+		} }
+		persona={ initialPersona }
+		tone={ initialTone }
```

Comment:
This change of order is my own doing. Especially when there are many properties/entries, I like sorting things alphabetically. This does not apply to function declarations of course 🙂